lib/engine/game/g_rolling_stock/game.rb

```diff
@@ -153,13 +155,28 @@ def setup
 
           @foreign_investor = Player.new(-1, 'Foreign Investor')
           @bank.spend(FOREIGN_START_CASH, @foreign_investor)
-          @cost_level = @company_stars[@company_deck[0].sym]
+          @cost_level = @company_stars[@company_deck[0]]
           @log << 'No cost of ownership'
           @cost_table = COST_OF_OWNERSHIP_RSS
+          @synergy_income = {}
+
+          # @corporations.find { |c| c.name == 'OS' }&.add_ability(GRollingStock::Ability::Overseas.new(type: :overseas))
+          self.class::CORPORATIONS.each do |corp|
+            next unless (custom = corp[:custom_ability])
+
+            klass = Engine::Ability::Base.type(custom)
+            ability = Object.const_get("Engine::Game::GRollingStock::Ability::#{klass}").new(type: custom.to_sym)
```

Comment:
don't do this

Reply:
What would you suggest? Perhaps I can specify the ability in custom_ability